internal/context/management_data.go

```diff
@@ -574,7 +574,7 @@ func GetNofificationUri(nfProfile models.NfProfile) []string {
 	if nfProfile.SNssais != nil {
 		var networkSliceFilter bson.M
 		var snssaisBsonArray bson.A
-		for _, snssai := range *nfProfile.SNssais {
+		for _, snssai := range nfProfile.SNssais {
```

Comment:
```suggestion
		for _, snssais := range nfProfile.SNssais {
```

Reply:
@andy89923 why?

Reply:
@yccodr sorry, I think this is a mistake 😂